src/Security/Member.php

```diff
@@ -254,14 +254,6 @@ class Member extends DataObject
      */
     private static $auto_login_token_lifetime = 172800;
 
-    /**
-     * Used to track whether {@link Member::changePassword} has made changed that need to be written. Used to prevent
-     * the write from calling changePassword again.
-     *
-     * @var bool
-     */
-    protected $passwordChangesToWrite = false;
```

Comment:
Potentially I need to restore this. I'd consider this not part of public API and it's no longer used in the new implementation